### PR TITLE
Snyk App Submission

### PR DESCRIPTION
**Pull request by andrewpowell**
Snyk for Tidbyt.  An app to display issue count for a Snyk project on Tidbyt.

### CODE REVIEW

**Comment by andrewpowell** [comment]
I have read the CLA Document and I hereby sign the CLA

File: apps/snyk/snyk.star [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+                                                color = "#D17D01",
+
+                                            ),
+                                            render.Text(
+                                                content = "High: %d" % high,
+                                                font = "tom-thumb",
+                                                color = "#CB4F19",
+                                            ),
+                                            render.Text(
+                                                content = "Crtical: %d" % critical,
```
Comment (rgkimball) [inline apps/snyk/snyk.star:147]: `Crtical`: Is this a typo?

Comment (andrewpowell) [reply apps/snyk/snyk.star]: Yes.  Fixed

**Comment by rohansingh** [comment]
Looks great! I added a minor change to put the logo in the file instead of fetching over HTTP.

**Pull request merged**